package.json

```diff
@@ -230,10 +229,10 @@
     "resin-doodles": "^0.1.1",
     "resin-image-fs": "^5.0.9",
     "resin-multibuild": "^4.4.2",
-    "resin-release": "^1.2.0",
+    "resin-release": "^1.2.2",
```

Comment:
This doesn't change the shrinkwrapped version, just updated the package.json to match that actual version that will be installed. This should also be replaced by balena-release @pdcastro / @srlowe which I believe only has the breaking change of requiring node 8+ but I don't really have the capability to test so it'd be great if one of you could handle it

Reply:
Ok, opened issue for that: https://github.com/balena-io/balena-cli/issues/1741